web/docs/Frontend Starters/Gatsby/Gatsby + WordPress/add-ons.md

```diff
@@ -0,0 +1,96 @@
+---
+id: 'gatsby-wordpress-add-ons'
+title: 'Add Ons'
+slug: '/frontend-starters/nextjs/gatsby/gatsby-wordpress/add-ons'
+sidebar_position: 5
+---
+
+### Advanced Custom Fields
+
+Advanced Custom Fields turns WordPress sites into a fully-fledged content
+management system by giving you all the tools to do more with your data.
+
+Advanced custom fields let you add, store, and display additional information
+about a piece of content in WordPress. This information can easily be assessed
+through GraphQL with simple adjustments to existing queries.
+
+This add-on uses GraphQL to bring in a custom `related content` data type that
+was created with Advanced Custom Fields. Your starter kit will have a related
+content section that displays at the bottom of post detail pages, if related
+posts are specified on the related WordPress post.
+
+#### Before You Begin
+
+- Install and activate the
+  [WP Advanced Custom Fields Plugin](https://wordpress.org/plugins/advanced-custom-fields/).
+
+- Install the
+  [Create Pantheon Decoupled Kit CLI](https://www.npmjs.com/package/create-pantheon-decoupled-kit/).
+
+#### Instillation Through CLI
```

Comment:
```suggestion
## Adding the `gatsby-wp-acf-addon`  to an Existing Project
```
'Installing' doesn't feel right to me, open to other options here. Also, this should either be a new section (h2), or be nested under a more generic heading like "Setup" or move "Usage" up to here as the generic section